.github/workflows/go.yml

```diff
@@ -36,7 +36,9 @@ jobs:
     steps:
       - uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: 1.20
+        env:
+          GO111MODULE: off
```

Comment:
`GO111MODULE` should not be used since something like Go 1.13-1.15.

Reply:
I desperately couldn't understand why the linter didn't work.